enh: add some more documentation for antivirus usage

max-nextcloud opened the pull request:
![grafik](https://github.com/nextcloud/documentation/assets/97337118/03cc5d7a-3884-4da7-8513-3ed43a4dfeb0)

max-nextcloud commented:
https://github.com/nextcloud/files_antivirus/pull/284 added the commands.

luka-nextcloud commented on admin_manual/configuration_server/antivirus_configuration.rst:
```diff
@@ -169,10 +169,55 @@ Executable
+
+::
+
+    occ files_antvirus:status [-v]
```
Type: should be `occ files_antivirus:status [-v]` (missing `i`) ;)

luka-nextcloud commented on admin_manual/configuration_server/antivirus_configuration.rst:
```diff
@@ -169,10 +169,55 @@ Executable
+
+::
+
+    occ files_antvirus:scan <path>
```
Type: should be `occ files_antivirus:scan <path>` (missing `i`) ;)

kesselb commented:
Hi, thanks for your nice pull request :+1: 

We have a different page for most of the occ commands: https://github.com/nextcloud/documentation/blob/master/admin_manual/configuration_server/occ_command.rst

Do you think it makes sense to move them over there?

max-nextcloud commented:
I tried to include all feedback:
* typos fixed
* added occ commands also on the occ command page

I also changed the occ commands in the antivirus docs to include `sudo -u www-data php ` to match the ones in the occ docs.

@luka-nextcloud @kesselb could you have another look?

max-nextcloud commented:
/backport stable28
